configs/local/portal-ext.properties

```diff
@@ -6,6 +6,17 @@
 #jdbc.default.username=root
 #jdbc.default.password=
 
+#
+# Feature flags
+#
+
+# Enable the import/export center
+feature.flag.COMMERCE-8087=true
```

Comment:
@rotty3000  I added the feature flag, so import/export center should be visible for wed's session.